.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-  - 1.10.x
+  - 1.x
```

Comment:
Is there a way to exclude pre-releases?

Reply:
"stable" should do the job as travis use gimme
https://github.com/travis-ci/gimme
I'll try it...